### DISCUSSION TITLE
Any guide for update the controlplane.yaml and worker.yaml talos version?

### DISCUSSION THREAD

**Discussion by daluntw** [siderolabs/talos#7818] [Q&A]
After the cluster updated, the image version in yaml files won't be upgrade 

and will be the old version when starting a new controlplane / worker

Is there any guide / tool that can update these two YAML file? (version & new feature default?)

**Comment by smira** [comment]
you can use [patching](https://www.talos.dev/v1.5/talos-guides/configuration/patching/#configuration-patching-with-talosctl-cli) [flow](https://www.talos.dev/v1.5/introduction/prodnotes/#configure-talos) with `talosctl`, or you can use higher-level orchestration tool like [Omni](https://www.siderolabs.com/platform/saas-for-kubernetes/) to take care of that for you
